src/Avalonia.Controls/CalendarDatePicker/CalendarDatePicker.cs

```diff
@@ -918,5 +918,13 @@ private static bool IsValidDateFormatString(string formatString)
         {
             return !string.IsNullOrWhiteSpace(formatString);
         }
+
+        /// <summary>
+        /// Clear SelectedDate
```

Comment:
So a few nits here.

 1. It's usually best to do a <see cref"SelectedDate" /> link when referencing a property directly. This ensures it is picked up in renaming and search.
 2. Wording probably should be a full sentence `Clears the <see cref"SelectedDate" />.`
 3. Missing period.